### ISSUE TITLE
Grey maximize drawer button on white background fails requirement

### ISSUE THREAD

**Issue by shanelin1** [eBay/ebayui-core#1635]
# Bug Report
ebay-drawer - maximize drawer button

## Description
Expected Result
Interactive elements have contrast ratio of at least 3:1 against the background.

Actual Result
The expand/collapse button is grey #A2A2A2 with contrast of 2.55:1 against the white (#FFFFFF) background. Visually impaired users may not be able to see the button.

HTML:
<button aria-label="Maximize Drawer" class="drawer-dialog__handle"></button>

CSS:
.drawer-dialog__handle::after

{ background-color: var(--dialog-handle-color,var(--color-grey4,#a2a2a2)); border-radius: 5px; content: ""; display: block; height: 3px; width: 20px; }
Failed WCAG 2.0 checkpoint(s)
1.4.3

Suggested resolution(s)
Use a darker grey of grey that will have contrast ratio of at least 3:1 against the background. https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

## Workaround
n/a

**Comment by shanelin1** [comment]
moved to ebay skin: https://github.com/eBay/skin/issues/1686
